policyengine_us/parameters/gov/states/ar/tax/income/gross_income/capital_gains/rate.yaml

```diff
@@ -0,0 +1,19 @@
+description: Arkansas taxes this percentage of long-term capital gains.
+metadata:
+  unit: /1
+  period: year
+  label: Arkansas long-term capital gains tax percentage
+  reference:
+    - title: Arkansas Code Title 26. Taxation § 26-51-815. Capital gains and losses (b) (2)
+      href: https://codes.findlaw.com/ar/title-26-taxation/ar-code-sect-26-51-815.html
+    - title: Arkansas 2021 AR1000D 
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2021_AR1000D_CapitalGains.pdf#page=1
+    - title: Arkansas 2022 AR1000D 
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000D_CapitalGains.pdf#page=1
+    - title: Arkansas 2023 AR1000D 
+      href: https://www.taxformfinder.org/forms/2023/2023-arkansas-form-ar1000d.pdf#page=1
+
+values:
+  2015-01-01: 0.5
+  2015-02-01: 0.45
```

Comment:
This is the portion exempt, not taxed. Flip to match the legal code.

Reply:
flipped, to represent legal code